lib/pbench/cli/agent/options.py

```diff
@@ -19,13 +19,13 @@ def callback(ctx, param, value):
     return click.option(
         "-C",
         "--config",
+        required=True,
         envvar="_PBENCH_AGENT_CONFIG",
-        type=click.Path(exists=True),
+        type=click.Path(exists=True, readable=True),
         callback=callback,
         expose_value=False,
         help=(
-            "Path to a pbench-agent config. If provided pbench will load "
-            "this config file first. By default is looking for config in "
-            "'_PBENCH_AGENT_CONFIG' envrionment variable."
+            "Path to a pbench-agent configuration file (may be specified "
+            "via the '_PBENCH_AGENT_CONFIG' environment variable, instead)"
```

Comment:
I like "defaults to", which I think more clearly implies that the environment variable only matters if you don't use `--config|-C`.

Reply:
Tweaked.